File: src/main/java/com/my/admin/app/commerce/model/document/OrderType.java
package com.my.admin.app.commerce.model.document;

import lombok.Data;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import java.io.Serializable;

@Document(collection = "order_types")
@Data
public class OrderType implements Serializable {

    private static final long serialVersionUID = 1L;

    @Id
    private Long id;

    private String name;

    private String description;
}

File: src/main/kotlin/com/bk/reactive/app/commerce/admin/my/model/document/OrderType.kt
package com.bk.reactive.app.commerce.admin.my.model.document

import org.springframework.data.annotation.Id
import org.springframework.data.mongodb.core.mapping.Document

@Document(collection = "order_types")
data class OrderType(
    @Id
    val id: String,
    val name: String,
    val description: String
)